### ISSUE TITLE
Implement Responsive Design for Website

### ISSUE THREAD

**Issue by abhijeetnishal** [group-patronage/group-patronage#18]
### Description:
Currently, the website lacks responsiveness, which hinders accessibility and usability for users accessing it from mobile devices or different screen resolutions.

To address this issue, I implement responsive design to make this website adapt and display optimally on all devices. This improvement will enhance the user experience, attract more visitors, and ultimately contribute to the project's growth and success.

@LakshaySK106  Assign me this issue under GSSOC2023

**Comment by LakshaySK106** [comment]
I believe the website is quite responsive in nature already. Nevertheless, you may work on it. I'll assign it to you.

**Comment by abhijeetnishal** [comment]
In smaller screen, it is not responsive. 
Screenshot:
<img width="451" alt="Screenshot 2023-05-22 at 12 17 51 AM" src="https://github.com/group-patronage/group-patronage/assets/83833440/2945cae2-bbbc-4424-b1ee-6a2d3efe2879">

**Comment by LakshaySK106** [comment]
Yes, you can fix it.
